plotter/boxplot.go

```diff
@@ -206,13 +212,19 @@ func (b *BoxPlot) Plot(c draw.Canvas, plt *plot.Plot) {
 	aLow := trY(b.AdjLow)
 	aHigh := trY(b.AdjHigh)
 
-	box := c.ClipLinesY([]vg.Point{
+	pts := []vg.Point{
 		{X: x - b.Width/2, Y: q1},
 		{X: x - b.Width/2, Y: q3},
 		{X: x + b.Width/2, Y: q3},
 		{X: x + b.Width/2, Y: q1},
 		{X: x - b.Width/2 - b.BoxStyle.Width/2, Y: q1},
-	})
+	}
+
+	box := c.ClipLinesY(pts)
+
+	if b.FillColor != nil {
+		c.FillPolygon(b.FillColor, c.ClipPolygonY(pts))
+	}
 	c.StrokeLines(b.BoxStyle, box...)
```

Comment:
I think this will read better as a single stanza, please remove the blank lines.